src/libs/window/include/os_window.hpp

```diff
@@ -38,6 +38,8 @@ class OSWindow
     virtual int Width() const = 0;
     //! Current height of window
     virtual int Height() const = 0;
+    //! Current window size
+    virtual void WindowSize(int *width, int *height) const = 0;
```

Comment:
Could you return a tuple or pair or struct instead of args?

Reply:
Changed to struct.